src/main/java/com/academy/terai/Model/Application.java

```diff
@@ -1,12 +1,209 @@
 package com.academy.terai.Model;
 
+import org.bson.types.Binary;
 import org.springframework.data.annotation.Id;
+import org.springframework.data.mongodb.core.index.Indexed;
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.format.annotation.DateTimeFormat;
 
+import javax.persistence.Entity;
+import javax.validation.constraints.Email;
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import javax.validation.constraints.Size;
+import java.util.Date;
+@Entity
 @Document(collection = "application")
 public class Application {
     @Id
     private String id;
 
+    @NotBlank(message = "Vardas negali būti tuščias")
+    @Size(max = 32, message = "Vardas negali būti ilgesnis nei 32 simboliai")
```

Comment:
Really detailed response messages, please check with Kazimieras regarding a proper place for a validation messages. I'd advise having all the validation in the Frontend so that these ones would never get triggered. However, if because of the frontend's bug it'd - it could be more abstract. However, if Kazimieras is OK with it, I don't mind keeping the in here either.